Lib/tkinter/__init__.py

```diff
@@ -763,15 +763,31 @@ def after_cancel(self, id):
 
         Identifier returned by after or after_idle must be
         given as first parameter."""
+        if id is None:
+            return
```

Comment:
Please see my following comment with the Tcl code.  I added this because I'm not sure if the original comments about the tuple changing were accurate.  I think it was a result of id being None vs it having a value.